model.py

```diff
@@ -20,7 +20,8 @@
 
 from enum import Enum
 import tkinter as tk
-import c31Geometry.c31Geometry2 as geo # type: ignore
+import c31Geometry.c31Geometry2 as Geo  # type: ignore
```

Comment:
```suggestion
import c31Geometry.c31Geometry2 as geo  # type: ignore
```
Les modules ne doivent pas avoir de majuscule (ex: `import tkinter as tk`)